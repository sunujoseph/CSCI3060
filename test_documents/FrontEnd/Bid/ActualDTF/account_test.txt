04 userSS                    item2 admin1          000100
00 admin1          AA 000150.78
04 admin1                    item1 userFS          000200
00 userFS          FS 111111.11
00 userBS          BS 123456.78
00 userSS          SS 222222.22
