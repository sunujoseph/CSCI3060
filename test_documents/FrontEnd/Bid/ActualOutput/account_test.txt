Enter Username: 
Enter Item: 
Enter Seller Username: 
Enter Bid: 
Error: Invalid Input
Enter Username: 
Enter Item: 
Enter Seller Username: 
Enter Bid: 
Error: Invalid Input
Enter Username: 
Error: You Do Not Have Privileges To Perform This Transaction
Error: Invalid Command
Error: Invalid Command
Error: Invalid Command
Error: Invalid Input
Enter Username: 
Error: You Do Not Have Privileges To Perform This Transaction
