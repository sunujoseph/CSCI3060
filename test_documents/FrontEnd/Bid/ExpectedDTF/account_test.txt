04 item2               userSS          admin1          100.00
00 admin1          AA 000150.78
04 item1               admin1          userFS          200.00
00 userFS          FS 111111.11
04 item 3              userFS          userBS          300.45
00 userBS          BS 123456.78
00 userSS          SS 222222.22