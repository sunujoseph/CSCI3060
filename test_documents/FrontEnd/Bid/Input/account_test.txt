login
admin1
bid
userSS
item2
100
logout

login
userFS
bid
admin1
item1
200
logout

login
userBS
bid
userFS
item3
300.45
logout

login
userSS
bid
logout
