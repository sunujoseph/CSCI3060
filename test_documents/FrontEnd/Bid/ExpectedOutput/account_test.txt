Enter Username: 
Enter Item Name: 
Enter Seller Username: 
Current Highest Bid Is: 012.34
Enter New Bid: 

Enter Username: 
Enter Item Name: 
Enter Seller Username: 
Current Highest Bid Is: 134.00
Enter New Bid: 

Enter Username: 
Enter Item Name: 
Enter Seller Username: 
Current Highest Bid Is: 100.00
Enter New Bid: 

Enter Username: 
Error: You Do Not Have Privileges To Perform This Transaction